packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+import { Badge } from 'terra-clinical-item-view/package.json?dev-site-package';
+import { Notice } from "@cerner/terra-docs";
+import ItemViewWithHeader from '../example/ItemViewWithHeader?dev-site-example';
+import ItemViewTwoColumnWithHeader from '../example/ItemViewTwoColumnWithHeader?dev-site-example';
+
+<Badge />
+
+# Accessibility Guide for Terra Clinical Item View
+
+## Why is this important?
+
+Terra Clinical Item View allows you to create a view filled with information in the form of Clinical Item Displays. Improper usage of this component may prevent end users from understanding content and necessary context to interact with the page. 
+
+## Accessibility Considerations
+
+#### General notes
+- **Related information should be presented as close to each other as possible.** It is best practice to keep related information above or below its related counterparts.
+  - For example - presenting related information across from each other in the two column view can make it more difficult for users to understand that the information is related.
+Reading by column from top to bottom is the more common way information is consumed visually, so when presenting this information by row between the two columns it might be confusing for both sighted and non-sighted users.
+- When passing in displays, keep in mind that only 8 can be used within a view at a time. Any additional displays passed in will be ignored.
+- Displays are also split between two columns with every other display being moved to the right hand column.
+
+----
+
+### Code Considerations
+
+#### Headings
+
+<Notice variant="important" ariaLevel="3">
+
+It is **strongly** suggested that consumers add a heading element above their implementation of the Clinical Item View component. 
+
+The Clinical Item View structure consists of an unordered list filled with list items, it is vital we include a header to ensure that the list of displays have proper context within the webpage.
+For example - if an Item View contains contact information for a patient, you should add a header above the Item View that says something like 'Contact Information' or 'Patient Contact Information'.
+
+**A header not only acts as a label for the information being communicated to the end users, but also helps with visual and screenreader navigation throughout a page.**
+To best implement this header, consumers can use the Terra Core Heading component found [here](https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/heading/about)
```

Comment:
Terra Core is an internal name. We should use the name of the component directly.
```suggestion
The recommended way to implement this header is to utilize [Terra Heading](https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/heading/about)
```

Reply:
https://github.com/cerner/terra-clinical/pull/916/commits/5117762dc6b5ef7e8b4056b22c0a99646b862df7#diff-bc5256d2671d1c03594dbeb9ddfffd9514706650fe9a7a330c26c0c02d080e10R37-R38